### PR TITLE
BSHIFT-386: Added new error message

### PR DESCRIPTION
**Pull request by narrow89**
Added new error message for bitso shift share wire instructions feature

### CODE REVIEW

**Review by viccomx — COMMENTED** [review]
Can you please add context on when and how this error code will be used?

**Comment by narrow89** [comment]
> Can you please add context on when and how this error code will be used?

Sure. We are creating a new endpoint that will be used to dispatch an email with the circle account wire instructions. This is a generic error that will be used in case something wrong happen during this process. Some of those possibles scenarios are:
* The list of recipients contains an invalid email
* The mail service is down
* The circle service is down

Refer to: https://github.com/bitsoex/bitsoex/pull/5418

File: source/index.html.md [hunk 1]
```diff
@@ -291,6 +291,7 @@ error categories, the last two digits define specific errors.
 * 1004: Deprecated functionality
 * 1005: Invalid operation
 * 1006: No content for given mobile client name (204 code)
+* 1007: Could not share wire instructions
```
Comment (thenewversion) [inline source/index.html.md:294]: As discussed in the engineering channel - I would mark this as `Internal use` and document it somewhere else (TBC)

Comment (narrow89) [reply source/index.html.md]: Done -> 42d44d9

**Review by thenewversion — APPROVED** [review]

**Pull request merged**